latest/Dockerfile

```diff
@@ -36,10 +36,7 @@ RUN set -x \
  && wget https://bootstrap.pypa.io/get-pip.py -O- | python3 - \
  && pip install -U setuptools pip pipenv \
     # Install Python dependencies.
- && pipenv install --system --deploy \
-    # Clean up
- && python3 -m pip uninstall -y pip pipenv setuptools \
- && apk del --purge .build-deps
+ && pipenv install --system --deploy
```

Comment:
This image is meant to remain as small as possible and build dependencies are files that shouldn't be necessary at runtime. Please keep them.

Reply:
In some cases they are necessary: https://stackoverflow.com/questions/7446187/no-module-named-pkg-resources

Reply:
We could try leaving in only setuptools though. I don't recall if I tried that or if that caused issues. It was simpler to not remove things that may break the functionality, especially since this repo has no integration tests.

Reply:
Keeping the image from being broken is more important than it being small, I assumed :)